src/main/java/org/orecruncher/dsurround/lib/biome/BiomeUtils.java

```diff
@@ -2,16 +2,16 @@
 
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.BuiltinRegistries;
 import net.minecraft.world.World;
 import net.minecraft.world.biome.Biome;
 import net.minecraft.world.biome.BiomeKeys;
-import net.minecraft.world.biome.BuiltinBiomes;
 
 public final class BiomeUtils {
 
     // Default to the VOID because PLAINS has data associated with it
     public static final Identifier DEFAULT_ID = BiomeKeys.THE_VOID.getValue();
-    public static final Biome DEFAULT_BIOME = BuiltinBiomes.THE_VOID;
+    public static final Biome DEFAULT_BIOME = BuiltinRegistries.BIOME.get(BiomeKeys.THE_VOID);
```

Comment:
Is this compatible with the datapack system? Should it be a getter that runs the lookup on-demand instead?

Reply:
Why not? I am pretty sure if it isn't compatible now, it also wasn't compatible before.
However, if you want, I can transform it to a getter